packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/demos/Chatbot.tsx

```diff
@@ -281,14 +283,35 @@ export const ChatbotDemo: React.FunctionComponent = () => {
     </>
   );
 
+  const handleSkipToContent = (e) => {
+    e.preventDefault();
+    if (displayMode === ChatbotDisplayMode.default) {
+      if (!chatbotVisible && toggleRef.current) {
+        toggleRef.current.focus();
+      }
+      if (chatbotVisible && chatbotRef.current) {
+        chatbotRef.current.focus();
+      }
+    } else {
+      if (chatbotRef.current) {
+        chatbotRef.current.focus();
+      }
+    }
+  };
+
   return (
     <>
+      <SkipToContent onClick={handleSkipToContent} href="#">
+        Skip to chatbot
```

Comment:
```suggestion
        Skip to chatbot
```

I might be getting confused here. I didn't see this text/label anywhere in the demo -- is this different than the "back to bottom" button? Or does this text not show in the UI anywhere

Reply:
It is a button. It only appears if you've tabbed into the main window. Rather than going to the bottom, it skips any intervening content between it and the target you set. It takes you straight to the toggle, or the chatbot region itself, depending on which example.

* Default takes you to the toggle when closed and chatbot region when open.
* Docked and fullscreen take you to the chatbot region when open.
* Embedded takes you to the chatbot region.

<img width="1487" alt="Screenshot 2024-11-06 at 6 45 25 PM" src="https://github.com/user-attachments/assets/40ac8fa6-8632-4ebd-acce-d806e07deff2">

Reply:
ohhh okay I didn't tab into the window -- my mistake 😆 was looking at the wrong thing lol